docker-compose.monitor-client.commercial.yml

```diff
@@ -4,5 +4,12 @@ services:
     #
     # monitor-client
     #
-    monitor-client:
+    mender-client:
```

Comment:
This will conflict with the `release_tool` querying (and eventually the next release). I am taking note on my TODO to think how to address it.

Reply:
> This will conflict with the `release_tool` querying (and eventually the next release). I am taking note on my TODO to think how to address it.

thank you!

Reply:
@kacf do you think that can be the reason behind the "old code" problem we discussed today? (I am now looking for every possible hint)

Reply:
No, I don't think so, because there already exists an instance of this [here](https://github.com/mendersoftware/integration/blob/master/docker-compose.docker-client.addons.yml#L4-L6).

Reply:
I forgot to update here. I was wrong on my original assessment and this is not a problem (if any, if would have been a problem at the time of merging the feature branch).

Reply:
It came to my mind now: what we were missing is an update in `components-maps.yml`! See PR:
* https://github.com/mendersoftware/integration/pull/1556